ewcfg.py

```diff
@@ -3034,6 +3034,57 @@ def wef_scythe(ctn = None):
+		str_desc = "A cold, refreshing can of Khaotic Killiflower flavored FUCK ENERGY. You can occasionally feel rumbles from inside it, the spirit of the drink itself begging to be released from the thin metal sarcophagus that barely contains it. You admire it’s stunningly attractive logo and general graphic design, before flipping over to read the back of the can.\n\n*FUCK ENERGY\n“GET FUCKED.™”\nMake no mistake - FUCK ENERGY is not your grandma's run-of-the-mill pissy baby fucker fapper limp, lame liquid masquerading as a psychotic psycadellic or performance-enhancing exlir. FUCK ENERGY is the real deal. From the moment you bought this energy drink, your fate was sealed, cursed. Reality itself has been rewritten, and your destiny decided. Your body's natural limits and basic inhibitions will be completely and utterly pulverized, anhilliated, and grinded into dust to be scavenged by us to be imbued into the next incarnation of the very instrument of your destruction. That's right. Every FUCK ENERGY is infused, steeped in the atomized souls of our unprepared consumers. You will contribute to this vicious cycle, at a near molecular level your very consciousness will be ripped apart and sold into slavery. Your master? FUCK ENERGY. Every drop of FUCK ENERGY has been rigorously tested to systematically attack you, shutting down entire unwanted bodily functions. Your organs will be forcefully transformed into top-of-the-line computer parts, hand picked by a cruel computer science major to maximize the fidelity of his foreign language visual erotica. Your brain will be overclocked, your heart pushed past all previous extremes, and without an internal fan to cool it down. Don't be surprised if the gradient of rainbow lights illuminating your surroundings from deep inside your body attracts ridicule and social ostracization. You will be a pariah, a being of pure addreillne and a martyr for dopamine. You will be consumed by the abstract idea energy. But, it won't be abstract to you. You will understand energy more than any over living creature on this planet. Now go, open this quite literal Pandora's Box. Escaping your purpose is impossible. What are you waiting for? Are you scared? GET FUCKED.\nNot intended for use in people above the age of 18.*",
```

Comment:
psycadellic -> psychedelic
exlir -> elixir
anhilliated -> annihilated
grinded -> ground
addreillne -> adrenaline
the abstract idea energy -> the abstract idea of energy